docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+# Battery Monitor
+
+You can access information related to power consumption by applications or by hardware resource on a battery-powered device for a certain duration of time. The value is returened as a percentage of the total batttery capacity.
+
+The power consumption for an application is calculated by accumulating, the usage of various individual resource used by the application.
+
+The Battery Monitor provides following provisions to fetch the Battery Usage:
+
+- Getting Battery usage information for a particular application-
+
+  You can [retrieve information for an application](#appusage_get) by specifing its application id, the [resource](#resource_key) id(s) and the time [duration](#duration_key).
+
+- Getting Battery usage information for a particular resource-
+
+  You can [retrieve information for a resource](#resourceusage_get)(single or multiple) by specifing the [resource](#resource_key) id(s) and the time [duration](#duration_key).
+
+## Prerequisites
+
+To enable your application to use the Battery Monitor functionality:
+
+1. To use the functions and data types of the Battery Monitor API (in [wearble](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__MONITOR__MODULE.html) applications), include the `<battery_monitor.h>` header file in your application:
+
+   ```
+   #include <battery_monitor.h>
+   ```
+
+2. The application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+   ```
+   <privileges>
+      <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+   </privileges>
+   ```
+
+<a name="appusage_get"></a>
+## Getting Battery usage information for a particular application
+
+Some Battery Usage information contain combination of Application Id, Resource Id and time duration.
+
+1. Fetching information for an application for a particular resource over certain duration-
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, for example:
```

Comment:
The `battery_monitor_resource_id_e` and the `battery_monitor_duration_type_e` enums can be used to provide information related to a specific resource and a particular duration type respectively, as shown in the following code: